app/services/usps_confirmation_uploader.rb

```diff
@@ -15,10 +15,9 @@ def generate_export(confirmations)
   end
 
   def upload_export(export)
+    io = StringIO.new(export)
     Net::SFTP.start(*sftp_config) do |sftp|
-      sftp.file.open(remote_path, 'w') do |file|
-        file.write(export)
-      end
+      sftp.upload!(io, remote_path)
```

Comment:
Can we set a reasonable timeout? https://github.com/ankane/the-ultimate-guide-to-ruby-timeouts#net-sftp